posthog/models/action/action.py

```diff
@@ -12,6 +12,23 @@
 from posthog.redis import get_client
 
 
+ActionStepMatching = Literal["contains", "regex", "exact"]
+
+
+@dataclass
+class ActionStepJSON:
+    tag_name: str
```

Comment:
If you're building a new thing, we might not want `tag_name` on it. We discussed deprecating it here due to subtle bugs this causes: https://github.com/PostHog/posthog/pull/21681

The tag name is useful when displaying a line like "clicked a div" in the activity page, but actions should only use selectors for clarity, since the tag_name is not exposed in the UI when editing an action.

Reply:
Not new - just moving it. I'll tag you for review though and sync with you on it